eva/parser/parser_visitor/_table_sources.py

```diff
@@ -176,3 +186,12 @@ def visitFromClause(self, ctx: evaql_parser.FromClauseContext):
             where_clause = self.visit(ctx.whereExpr)
 
         return {"from": from_table, "where": where_clause}
+
+    def visitAliasClause(self, ctx: evaql_parser.AliasClauseContext):
+        alias_name = self.visit(ctx.uid())
+        column_list = []
+        if ctx.uidList():
+            column_list = self.visit(ctx.uidList())
```

Comment:
I feel it becomes harder over time to figure out which `visit` function it actually calls. Any idea to improve the readability?

Reply:
It is actually quite intuitive, thanks to antlr.
`self.visit(ctx.uidList()) -> visitUidList`

Reply:
I see it is somehow in the `_insert_statements.py`. I missed it.